bin/linguist

```diff
@@ -16,14 +17,17 @@ end
 
 ARGV.shift
 breakdown = true if ARGV[0] == "--breakdown"
+json_breakdown = true if ARGV[0] == "--json"
```

Comment:
No objections. I designed them to be mutually exclusive because the current command line argument handling is rudimental and I would have to rewrite it with two possible options without knowing Ruby. Actually, the way --breakdown is handled already sucks IMHO.